go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -91,16 +91,17 @@ def update_files # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
             # Replace full paths with path hashes in the go.mod
             substitute_all(substitutions)
 
-            # Set the stubbed replace directives
-            update_go_mod(dependencies)
-
-            # Then run `go get` to pick up other changes to the file caused by
-            # the upgrade
-            run_go_get
+            # TODO as far as I can tell, this array will only ever have one value, but we should assert that somehow
+            # and blow up if it's ever not true... I'm not a rubyist, so not sure how best to handle?
```

Comment:
Help please?

Reply:
Tbh I'm not entirely sure if this is only ever 1 dependency or multiple in some cases. Can we run `go get` for each dependency in the array? That seems safest.

Reply:
Oh, it turns out `go get` accepts multiple package arguments, so that makes life much simpler/more intuitive: https://golang.org/pkg/cmd/go/#hdr-Add_dependencies_to_current_module_and_install_them